### ISSUE TITLE
change to radio buttons

### ISSUE THREAD

**Issue by satra** [Remi-Gau/cobidas-PET#5]
instead of dropdowns, perhaps changing to radio buttons may be useful when number of items is say less than 10.

should also be done for cobidas-main.

**Comment by Remi-Gau** [comment]
Yup I agree but see https://github.com/Remi-Gau/COBIDAS_chckls/issues/132

So maybe something to add to the to-do list on the UI front.
